docs/src/installing.md

```diff
@@ -2,24 +2,24 @@
 
 There are a few different ways to install Tealdeer:
 
-- Through [packet managers](#packet-managers)
+- Through [package managers](#package-managers)
 - Through [static binaries](#static-binaries-linux)
 - Through [cargo install](#through-cargo-install)
 - By [building from source](#build-from-source)
 
 Additionally, when not using system packages, you can [manually install
 autocompletions](#autocompletion).
 
-## Packet Managers
+## Package Managers
 
 Tealdeer has been added to a few package managers:
 
 - Arch Linux AUR: [`tealdeer`](https://aur.archlinux.org/packages/tealdeer/),
   [`tealdeer-bin`](https://aur.archlinux.org/packages/tealdeer-bin/) or
   [`tealdeer-git`](https://aur.archlinux.org/packages/tealdeer-git/)
+- Brew [`tealdeer`](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/tealdeer.rb)
```

Comment:
Sorry for nitpicking, but I think most people know the project under the name "Homebrew" (even if the package manager itself is called "brew"). At least the project refers to itself as "Homebrew". Maybe renaming it would be better? (Also, if you rename, sort alphabetically again.)